pxr/usd/usdGeom/camera.cpp

```diff
@@ -581,4 +653,23 @@ UsdGeomCamera::SetFromCamera(const GfCamera &camera, const UsdTimeCode &time)
     GetFocusDistanceAttr().Set(camera.GetFocusDistance(), time);
 }
 
+float
+UsdGeomCamera::GetExposureScale(UsdTimeCode time) const 
```

Comment:
Is there any reasonable way to make this function a pass-through to the "exposure" attribute if the other attributes are un-authored?  And is "exposure" (here exposureCompensation) just a non-physical modifier that you'd want to leave unauthored if you're trying to use the new controls?

Reply:
It should already do that: it defaults all the others so if they're not authored it will only take the exposure attribute into account (the defaults for the others combine to do nothing)